EntityFramework.Utilities/EntityFramework.Utilities/SqlQueryProvider.cs

```diff
@@ -85,6 +86,47 @@ public void InsertItems<T>(IEnumerable<T> items, string schema, string tableName
 			}
 		}
 
+		public async Task InsertItemsAsync<T>(IEnumerable<T> items, string schema, string tableName, IList<ColumnMapping> properties, DbConnection storeConnection, int? batchSize, int? executeTimeout, SqlBulkCopyOptions copyOptions, DbTransaction transaction)
+		{
+			var reader = new EFDataReader<T>(items, properties);
+#if (NETSTANDARD2_1 || NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_0_OR_GREATER)
```

Comment:
I'm copying some of these things over to my refactor. I was wondering about this right here. Wouldn't `#if (NETSTANDARD2_1_OR_GREATER)` be sufficient?

Reply:
It turns out that the `*_OR_GREATER` is only supported on the .NET 5 SDK and higher, and given that this library is a little old, I wasn't sure what SDKs people might be using to develop this library, so I threw the extra flag in just to be safe. In modern Microsoft libraries, they definitely don't bother worrying about old SDKs. Totally up to you if you include it 🙂